tests/Tests/500__jupyterhub/test-jupyterlab-notebook.robot

```diff
@@ -54,3 +54,12 @@ Can Launch Python3 Smoke Test Notebook
   #Get the text of the last output cell
   ${output} =  Get Text  (//div[contains(@class,"jp-OutputArea-output")])[last()]
   Should Not Match  ${output}  ERROR*
+
+Verify Proper image is provided and server starts
```

Comment:
According to the description of RHODS-1460, in order to better explain the purpose of this test it could be renamed (here and in Polarion) to  `Verify A Default Image Is Provided And Starts Successfully`

[Documentation]    Verify that, if a user doesn't explicitly select any jupyter image, a default one is selected and it can be spawned successfully

Reply:
Updated.